modules/nw-ipfailover-configuration.adoc

```diff
@@ -159,27 +203,3 @@ spec:
 <11> The strategy for handling a new higher priority host. The default value is `preempt_delay 300`, which causes a Keepalived instance to take over a VIP after 5 minutes if a lower-priority master is holding the VIP.
 <12> The period, in seconds, that the check script is run. The default value is `2`.
 <13> Create the pull secret before creating the deployment, otherwise you will get an error when creating the deployment.
-////
```

Comment:
Confirmed that the commented out example has no major addition to the context of this file.